src/templates/stepper.html

```diff
@@ -182,57 +182,109 @@ <h4><span class="help float-right"> By clicking the <code>Previous State</code>
                     {% endfor %}
                 </div>
             </div>
-            <!-- This view lets you look at multiple tree views at once, for state by state. -->
-            <div class="tab-pane fade" id="table-view" role="tabpanel">
-
-
-                <p>
-                Sub-formulae are demarcated by their bounding boxes, which
-                indicate whether they are <span class="satformula">satisfied</span> or
-                                    <span class="unsatformula">unsatisfied</span>.
-                                   
-                </p>
-                <!-- Table view goes here -->
-                <table class="table table-bordered" id="stateTable">
-                    <thead>
-                        <tr>
-                            <th class="w-auto"></th>
-                            {% for ps in prefixstates %}
-                                <th></th>
-                            {% endfor %}
-                            {% if cyclestates %}
-                                <th class="cycle-state-col" colspan="{{ cyclestates|length }}">Cycle</th>
-                            {% endif %}
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <tr>
-                            <th class="w-auto">Trace State</th>
-                            {% for ps in prefixstates %}
-                                <td><pre>{{ ps.formattedTraceAssignment }}</pre></td>
-                            {% endfor %}
-                            {% for ps in cyclestates %}
-                                <td class="cycle-state-col"><pre>{{ ps.formattedTraceAssignment }}</pre></td>
-                            {% endfor %}
-                        </tr>
-                        <tr>
-                            <th class="w-auto">
-                                Formula
-                                </span>
-                            </th>
-                            {% for ps in prefixstates %}
-                                <td>
-                                    {{ ps.formulaAsHTML | safe }}
-                                </td>
-                            {% endfor %}
-                            {% for ps in cyclestates %}
-                                <td class="cycle-state-col">
-                                    {{ ps.formulaAsHTML | safe }}
-                                </td>
-                            {% endfor %}
-                        </tr>
-                    </tbody>
-                </table>
+            <!-- This view lets you look at multiple tree views at once, for state by state. -->
```

Comment:
Structure this so that we have all 3. views, and in the source you can tell which is where. Maybe delimit with comments or something?